### PR TITLE
Tiny Tweaks

### PR DESCRIPTION
**Pull request by Blazzike**

### CODE REVIEW

File: src/css/index.css [hunk 1]
```diff
@@ -8,11 +8,25 @@
...
+#section-why .col.l3 {
+  padding: 30px 20px;
+}
 i.benefit-icon {
   text-align: center;
   display: block;
   font-size: 10rem;
 }
+.jumbotron {
+  background-image: url(https://i.imgur.com/SJ0J0DX.png);
```
Comment (aikar) [inline src/css/index.css]: this needs to be hosted on our own domain. didn't notice kash did that :P

File: src/images/logo-marker.svg [hunk 2]
```diff
@@ -1,49 +1 @@
...
... -3,-1.31c-1.007,-0.667 -1.907,-2.647 -2.7,-5.94l-0.15,0c-4.373,9.687 -6.78,14.72 -7.22,15.1l-0.3,0l-0.75,-0.44l-0.45,0.15l-0.15,-0.15l-0.15,0.15l-0.3,-0.58l-0.3,0.29l-0.15,-0.15l-1.35,0.29c-0.22,-0.527 -0.52,-0.817 -0.9,-0.87l0,-0.58c0,-2.32 2.857,-8.903 8.57,-19.75c1.34,-4 2.45,-5.95 3.3,-5.95l1.73,-0.31Z" style="fill:#fff;fill-rule:nonzero;"/><path d="M241.71,18.75l1.09,0c2.69,0 4.61,0.76 5.77,2.26l0,1.35c0,1.92 -0.73,2.87 -2.19,2.87c-1.114,-0.308 -2.265,-0.459 -3.42,-0.45c-7.62,0 -12.86,3.507 -15.72,10.52l-0.32,1.52l0,0.45c0,1.507 1.04,2.457 3.12,2.85c1.692,0.337 3.415,0.488 5.14,0.45l2.82,0l0,0.31l-0.64,0l0,0.14c0.25,0 0.51,0.2 0.79,0.6l0.15,0l0,0.45l-1.7,0.16l0,0.15l0.3,0l-0.3,0.9l0.3,0l0,0.3l-0.3,0l0,0.15c0.028,-0.005 0.057,-0.007 0.086,-0.007c0.29,0 0.53,0.239 0.53,0.53c0,0.026 -0.002,0.052 -0.006,0.077l0.32,0l-0.17,-0.15l0,-0.14l0.17,-0.16l0.93,0l0,0.26l0.15,0.15l-0.15,0.16l-0.46,0l0,0.14l0.3,0.31l0.46,0l-0.15,-0.16l0,-0.15l0.47,-0.14l0.47,0.14c0,0.9 -3.63,1.353 -10.89,1.36c-4.58,0 -7.797,-2.157 -9.65,-6.47l0,-2c0,-4.667 3.267,-9.63 9.8,-14.89c3.808,-2.564 8.31,-3.904 12.9,-3.84Zm-3.43,22.39l0.93,0l0,0.3l-0.93,0l0,-0.3Z" style="fill:#fff;fill-rule:nonzero;"/></g></g></svg>
```
Comment (zachbr) [inline src/images/logo-marker.svg]: I'm going to have to go through this.
What was changed here and why?

Comment (Blazzike) [reply src/images/logo-marker.svg]: Just centering the logo and text to the document and to one another.

Comment (aikar) [reply src/images/logo-marker.svg]: he shifted the logo up more from the text itself is what I noticed, and compacted it.

**Pull request merged**